go.mod

```diff
@@ -23,6 +23,6 @@ require (
 	golang.org/x/crypto v0.7.0 // indirect
 	golang.org/x/net v0.8.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/protobuf v1.29.0 // indirect
```

Comment:
The protobuf dependency update does not appear in the changelog.

Reply:
We decided in project keeper to omit indirect/transitive dependencies like for Maven projects.
Should we change this?